client/.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_ENGINE_BACKEND_URL=https://engine.arkhn.org
+REACT_APP_HTTP_BACKEND_URL=http://localhost:4000
+REACT_APP_WS_BACKEND_URL=ws://localhost:4000
+REACT_APP_CLEANING_SCRIPTS_URI=http://localhost:5000
```

Comment:
Will be changed to 5001 (conflict port with fhir api)

Reply:
thanks, done.